src/com/jalasoft/search/controller/SearchCriteria.java

```diff
@@ -0,0 +1,321 @@
+        return this;
+    }
+
+    /**
+     * Returns the attribute value advanceSearch
+     *
+     * @return String
+     */
+    public String getAdvanceSearch() {
+        return advanceSearch;
+    }
+
+    /**
+     * Sets the attribute value fileName
+     *
+     * @return String
+     */
+    public SearchCriteria setAdvanceSearch(String advanceSearch) {
+        this.advanceSearch = advanceSearch;
+        return this;
+    }
+
+    /**
+     * Returns the attribute value contains
+     *
+     * @return String
+     */
+    public String getContains() {
+        return contains;
+    }
+
+    public SearchCriteria setContains(String contains) {
+        this.contains = contains;
+        return this;
+    }
+
+    /**
+     * Returns the attribute value inTitle
+     *
+     * @return boolean
+     */
+    public boolean getInTitle() {
+        return inTitle;
+    }
+
+    public SearchCriteria setInTitle(boolean inTitle) {
+        this.inTitle = inTitle;
+        return this;
+    }
+
+    /**
+     * Returns the attribute value insideFile
+     *
+     * @return boolean
+     */
+    public boolean getInsideFile() {
+        return insideFile;
+    }
+
+    public SearchCriteria setInsideFile(boolean insideFile) {
+        this.insideFile = insideFile;
+        return this;
+    }
+
+    /**
+     * Returns the attribute value ownerFile
+     *
+     * @return String
+     */
+    public String getOwnerFile() {
+        return ownerFile;
+    }
+
+    public SearchCriteria setOwnerFile(String ownerFile) {
+        this.ownerFile = ownerFile;
+        return this;
+    }
+
+    /**
+     * Returns the attribute value fileCreated
+     *
+     * @return String
+     */
+    public boolean getFileCreated() {
+        return fileCreated;
+    }
+
+    public SearchCriteria setFileCreated(boolean fileCreated) {
+        this.fileCreated = fileCreated;
+        return this;
+    }
+
+    /**
+     * Returns the attribute value fileModified
+     *
+     * @return boolean
+     */
+    public boolean getFileModified() {
+        return fileModified;
+    }
+
+    public SearchCriteria setFileModified(boolean fileModified) {
+        this.fileModified = fileModified;
+        return this;
+    }
+
+    /**
+     * Returns the attribute value fileAccessed
+     *
+     * @return boolean
+     */
+    public boolean getFileAccessed() {
+        return fileAccessed;
+    }
+
+    public SearchCriteria setFileAccessed(boolean fileAccessed) {
+        this.fileAccessed = fileAccessed;
+        return this;
+    }
+
+//    public Date getInitDate() {
+//        return initDate;
+//    }
+//
+//    public SearchCriteria setInitDate(Date initDate) {
+//        this.initDate = initDate;
+//        return this;
+//    }
+//
+//    public Date getFinalDate() {
+//        return finalDate;
+//    }
+//
+//    public SearchCriteria setFinalDate(Date finalDate) {
+//        this.finalDate = finalDate;
+//        return this;
+//    }
+
```

Comment:
Remove these líneas if these are not required in the code. At least these be blocked and waiting some implementation, then comment it

Reply:
the block commented was removed